scrapers/PervCity.yml

```diff
@@ -0,0 +1,20 @@
+name: "PervCity"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - pervcity.com/trailers/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $sceneinfo: //div[@class="videoInfo"]
+    scene:
+      Title: $sceneinfo/div[@class="infoHeader"]
+      Performers: 
+        Name: $sceneinfo //span[@class="tour_update_models"]/a
+      Details: $sceneinfo//p/text()

```

Comment:
A little more complex but doable due to the nice pervcity search
We take the title, replace spaces with `_` (to make sure an exact result is returned ) and feed it to the search script url
```yaml
      Date:
        selector: $sceneinfo/div[@class="infoHeader"]
        postProcess:
          - replace:
              - regex: \s
                with: "_"
              - regex: ^
                with: "https://pervcity.com/search.php?query="
          - subScraper:
              selector: //div[@class="category_listing_block"]//div[@class="date"]/text()
              postProcess:
                - parseDate: 01-02-2006
```